build-tools/Dockerfile.rhel7.runtime

```diff
@@ -0,0 +1,49 @@
+FROM registry.access.redhat.com/rhel-atomic
+
+LABEL name="f5networks/k8s-bigip-ctlr" \
+      vendor="F5 Networks" \
+### Required labels above - recommended below
```

Comment:
Maybe remove this comment, or indicate that the version should be passed in as an arg to docker build.